config/routes.rb

```diff
@@ -13,7 +13,9 @@
       end
     end
     resources :documents, only: %i[index new create]
-    resources :meetings, only: %i[index new create show edit update]
+    resources :meetings, only: %i[index new create show edit update] do
+      resources :meeting_participants, only: %i[new create]
+    end
```

Comment:
Pensando no conceito de [shallow nasting](https://guides.rubyonrails.org/routing.html#shallow-nesting), onde a é recomendado só aninhar as rotas quando for necessário o ter o id dos elemento pai, acho que as rotas de `meeting_participants` poderiam ficar dentro de `meeting`, porém fora de `project`. E no caso teriam dois `resources :meetings`, onde um ficaria com nada dentro do only. Ex:

```ruby
resources :meeting, only: %i[] do
  resources :meeting_participants, only: %i[new create]
end
```

Dessa forma muitas rotas podem ser simplificadas. Nos testes não seria mais necessário passar o `project` nas rotas.

Reply:
Modificado, obrigada!